config/application.rb

```diff
@@ -84,26 +84,33 @@ class Application < Rails::Application
     # Defaults to every folder in the app directory of the application.
     config.eager_load_paths = []
 
-    require_relative 'environments/patches/database_configuration'
+    # This must be done outside of initialization blocks
+    #   as Vmdb::Logging is needed very early
+    require 'vmdb/logging'
 
-    console do
-      Rails::ConsoleMethods.class_eval do
-        include Vmdb::ConsoleMethods
-      end
-    end
+    config.before_initialize do
+      require_relative 'environments/patches/database_configuration'
```

Comment:
I also wanted to change this to live in lib/patches, but I decided to punt on that as it's completely unrelated.